addons/mysql-exporter/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+name: mysql-exporter
+version: 0.0.1
+appVersion: v1.14.0
```

Comment:
What's appVersion ?

Reply:
We should have an appVersion field same as the helm chart. The version field only represents the addon.